casper/src/main/scala/coop/rchain/casper/dag/BlockDagKeyValueStorage.scala

```diff
@@ -327,4 +363,62 @@ object BlockDagKeyValueStorage {
       stores.deployPool,
       stores.invalidBlocks
     )
+
+  def deployStatus[F[_]: Sync](
```

Comment:
Why is this method defined here on companion object and not directly as member on BlockDagKeyValueStorage?

Reply:
I was thinking this is easier to test.